datadog-checks-tests-helper/setup.cfg

```diff
@@ -0,0 +1,7 @@
+[aliases]
```

Comment:
we wanna remove this file and merge it into tox.ini. Can you do this for us please?

Reply:
Yes, put this in `tox.ini`:

```
[flake8]
exclude = .eggs,.tox,build
max-line-length = 120
```